_posts/2022-07-20-developer-cli.markdown

```diff
@@ -106,6 +106,16 @@ As things might not always be clear, here are some tips to help you:
 * Workloads are run under the _flotta_ user. Workloads debugging needs to be done also as _flotta_ user on the device.
 
 ### Debug the device
+For registering a device, the operator should download certificates and set them the correct permissions:
+```shell
+# remove old certs, if exists
+$ sudo rm /tmp/*.pem
+# download new certs
+$ make get-certs
```

Comment:
pls specify that this command should be executed from the flotta-operator project

Reply:
replaced `# download new certs` with `# download new certs, from the flotta-operator project`